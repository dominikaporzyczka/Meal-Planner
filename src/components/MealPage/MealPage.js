import React, { Component } from 'react';

export class MealPage extends Component {
  render() {
    return (
      <div>MealPage</div>
    );
  }
};
