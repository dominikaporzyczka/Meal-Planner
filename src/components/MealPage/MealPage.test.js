import React from 'react';
import renderer from 'react-test-renderer';
import { MealPage } from './MealPage.js';

describe('MealPage', () => {
  it('should match snapshot', () => {
    const mealPage = renderer
      .create(<MealPage />)
      .toJSON();

    expect(mealPage).toMatchSnapshot();
  });
});
