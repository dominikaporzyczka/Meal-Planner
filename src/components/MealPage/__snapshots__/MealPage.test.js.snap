// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MealPage should match snapshot 1`] = `
<div>
  MealPage
</div>
`;
